modules/gdscript/gdscript.cpp

```diff
@@ -2387,6 +2581,44 @@ Ref<GDScript> GDScriptLanguage::get_orphan_subclass(const String &p_qualified_na
 	return Ref<GDScript>(Object::cast_to<GDScript>(obj));
 }
 
+Ref<GDScript> GDScriptLanguage::get_script_by_fully_qualified_name(const String &p_name) {
```

Comment:
@adamscott this technically fixes #65935, but it doesnt fix the underlying issue in gdtype_from_datatype and _update_exports, and I think its outside the scope of this pr. ive been bathing in subclass code for over a month now and i have a few PRs in the works to bring subclasses closer to the stability of a script file and id like to have the honor of fixing my own issue

Reply:
You meant #65953!

Reply:
I modified the description of my PR to add a note about the fix of #65953. I think your PR should be merged before or after mine (in order to use your fix.)

![Capture d’écran du 2022-11-10 08-43-19](https://user-images.githubusercontent.com/270928/201107581-56a5849e-a317-4363-9357-e017055a6e4c.png)